vcell-util/src/main/java/org/vcell/util/Origin.java

```diff
@@ -96,6 +86,20 @@ public double getZ() {
  * This method was created in VisualAge.
  * @return java.lang.String
  */
+
+/**
+ * For unit conversion Purposes
+ *
+ * @param scaleFactor the factor to scale by
+ */
+public void rescaleInPlace(double scaleFactor, int dimension) {
+	switch (dimension){
+		case 3: this.x *= scaleFactor; this.y *= scaleFactor; this.z *= scaleFactor; break;
+		case 2: this.x *= scaleFactor; this.y *= scaleFactor; break;
+		case 1: this.x *= scaleFactor; break;
+	}
+}
+

```

Comment:
this method is no longer needed - and so a bit confusing if left in. (same comment as Extent)